apps/payments/next/.env.production

```diff
@@ -1,11 +1,11 @@
 #
 # Set default variables for production environment
-# e.g. yarn nx run payments-next:build 
+# e.g. yarn nx run payments-next:build
 #
 
 # Auth
-AUTH__ISSUER_URL=
-AUTH__WELL_KNOWN_URL=
+AUTH__ISSUER_URL=http://localhost:3030
+AUTH__WELL_KNOWN_URL=http://localhost:3030/.well-known/openid-configuration
```

Comment:
Q: Should the .env.production env file have localhost domains? Or are those _for_ production since they're local to the k8s pods?